react/features/conference/components/native/Labels.js

```diff
@@ -200,14 +204,18 @@ class Labels extends AbstractLabels<Props, State> {
                             this._renderTranscribingLabel()
                         }
                     </TouchableOpacity>
-                    {/*
-                      * Emil, Lyubomir, Nichole, and Zoli said that the Labels
-                      * should not be rendered in Picture-in-Picture. Saul
-                      * argued that the recording Labels should be rendered. As
-                      * a temporary compromise, don't render the
-                      * VideoQualityLabel at least because it's not that
-                      * important.
-                      */
+                    <TouchableOpacity
```

Comment:
have you checked how it looks slike in PiP mode?

Reply:
Labels are not visible on the reduced UI.

Now with this said, i removes the unused reduced UI logic from the Labels component too

Reply:
The recording one is, that'ss what the (overly verbose) comment was about...